pkg/ruler/ruler.go

```diff
@@ -288,6 +290,16 @@ func newRuler(cfg Config, manager MultiTenantManager, reg prometheus.Registerer,
 			Name: "cortex_ruler_sync_rules_total",
 			Help: "Total number of times the ruler sync operation triggered.",
 		}, []string{"reason"}),
+
+		ruleGroupStoreLoadDuration: promauto.With(reg).NewGauge(prometheus.GaugeOpts{
```

Comment:
The default sync period is once every minute. I originally had it as histogram but was not sure if we care about percentile calculations or distribution analysis since this should be low volume metric